maven-plugin-tools-generators/src/main/resources/help-class-source.vm

```diff
@@ -1,13 +1,20 @@
+#if ( $useMaven4Api )
+#set ( $mojoException = "MojoException" )
+#set ( $logger = "logger" )
+#else
+#set ( $mojoException = "MojoExecutionException" )
+#set ( $logger = "getLog()" )
+#end
```

Comment:
Maybe separate file for v3 and v4.
in V4 we can drop javadoc tags

Reply:
Done, and rebased / squashed.